### PR TITLE
feat(fake-users): create fake user

### PR DESCRIPTION
**Pull request by Ferkalek**
feat(fake-users): create fake user

created components for create fake users and display them

### CODE REVIEW

File: src/app/users/fake-users/fake-user.interfase.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export interface IFakeUserDTO {
```
Comment (ArtemLunt) [inline src/app/users/fake-users/fake-user.interfase.ts:1]: Fake user?
Pls think of some more revelant naming.

Comment (ArtemLunt) [reply src/app/users/fake-users/fake-user.interfase.ts:1]: Smth like `Employees`, for example

File: src/app/users/fake-users/fake-users-list/fake-users-list.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import { IFakeUserDTO } from '../fake-user.interfase';
+
+@Component({
+  selector: 'app-fake-users-list',
+  templateUrl: './fake-users-list.component.html',
+  styleUrls: ['./fake-users-list.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class FakeUsersListComponent {
+  @Select(FakeUserState.fakeUsers) fakeUsers$: Observable<IFakeUserDTO[]>;
```
Comment (ArtemLunt) [inline src/app/users/fake-users/fake-users-list/fake-users-list.component.ts:14]: readonly.

File: src/app/users/fake-users/fake-user.interfase.ts (hunk 1 above)
Comment (Ferkalek) [reply src/app/users/fake-users/fake-user.interfase.ts]: done

File: src/app/users/fake-users/fake-users-list/fake-users-list.component.ts (hunk 2 above)
Comment (Ferkalek) [reply src/app/users/fake-users/fake-users-list/fake-users-list.component.ts]: added

**Pull request merged**